cores/esp32/esp32-hal-rgb-led.h

```diff
@@ -0,0 +1,22 @@
+#ifndef MAIN_ESP32_HAL_RGB_LED_H_
+#define MAIN_ESP32_HAL_RGB_LED_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "esp32-hal.h"
+/*
+#include "soc/soc_caps.h"
+#include "pins_arduino.h"
+*/
+
+//#ifdef BOARD_HAS_NEOPIXEL
+  void RGBLedWrite(uint8_t pin, uint8_t red_val, uint8_t green_val, uint8_t blue_val);
```

Comment:
please cleanup the commented includes and ifdefs. Arduino starts function names with lower case. I would suggest that you rename the function to `neopixelWrite`. `rgbLed` is very general and should be attributed to actual RGB Led with separate leads for each color (and controlled through PWM).